aws/api-servers.tf

```diff
@@ -53,3 +61,41 @@ resource "aws_elb" "api-int" {
     lb_protocol = "tcp"
   }
 }
+
+resource "aws_security_group" "tsuru_api_int_elb" {
+  name = "${var.env}-tsuru-api-int-elb"
+  description = "Tsuru API Internal ELB security group"
+  vpc_id = "${aws_vpc.default.id}"
+
+  ingress {
+    from_port       = 443
+    to_port         = 443
+    protocol        = "tcp"
+    security_groups = [
+      "${aws_security_group.gandalf.id}"
+    ]
+  }
+
+  tags = {
+    Name = "${var.env}-tsuru-api-int-elb"
+  }
+}
+
+resource "aws_security_group" "tsuru_api" {
+  name = "${var.env}-tsuru-api"
+  description = "Tsuru API security group"
+  vpc_id = "${aws_vpc.default.id}"
+
+  ingress {
+    from_port       = 443
+    to_port         = 443
+    protocol        = "tcp"
+    security_groups = [
+      "${aws_security_group.web.id}"
```

Comment:
The following issue does not come from the current PR but from before, but maybe we should takle it in the future.

Currently we have a [unique security group](https://github.com/alphagov/tsuru-terraform/pull/99/files#diff-c2eb5021b3f76e661aa1d2440598a977R27) 
for the API loadbalancers and the Routers ones. This means that the router ELB could reach the API if the node is added.

This is not a problem meanwhile we set the ELB backends in terraform, but we are considering autoregister the routers dynamically
using something like [elb-presence](https://github.com/coreos/elb-presence). 

If we do so, we should split the `web` security group and create one for `api` and other for `router`.